### PR TITLE
Login signup page

### PR DESCRIPTION
**Pull request by Manju-sree**
Added Login and Sign-up Page

Sign-up Page
- you can see sign-up page from where one can sign-up using first name, last name, email, password & confirm password.
- you can see  hide-show icon button that shows or hides  password.

Login Page
- you can see a login page from where you can login using email & password.

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    font-weight: 600;
+    text-align: center;
+    padding-bottom: 1rem;
+}
+
+.form-row {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 1rem;
+    /* width: 100%; */
```
Comment (bhargaviRdy) [inline css/login.css]: remove comments everywhere

File: pages/Login.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                <label for="Email address" class="label-style star-required-field">Email address</label>
+                <input type="text" name="username" placeholder="username@gmail.com" class="input-box" required>
+            </ul>
+            <ul class="form-row wd-100-pc mr-lt-3 display-flex-column">
+                <label for="password" class="label-style mr-label-pwd star-required-field">Password</label>
+                <span class="password-eye-icon wd-100-pc pos-rel">
+                    <input type="password" name="username" placeholder="**************" class="input-box " required/>
+                <button class="card-btn eye-icon-container pos-abs" >
+                <i class="fas fa-eye-slash qty-btn-color"></i>
+            </button>
```
Comment (bhargaviRdy) [inline pages/Login.html]: try to format the code

**Review by bhargaviRdy — APPROVED** [review]
the changes look good, suggested some minor fixes.

File: pages/Loginin.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <h1 class="login-form-title">Login</h1>
+        <form action="login-form" class="form-container display-flex-column">
+            <ul class="form-row display-flex-column">
+                <label for="Email address" class="email-label">Email address</label>
+                <input type="text" name="username" placeholder="username@gmail.com" class="input-box" required>
+            </ul>
+            <ul class="form-row form-row-pwd display-flex-column">
+                <label for="password" class="email-label email-label-pwd">Password</label>
+                <span class="password-eye-icon pos-rel">
+                    <input type="password" name="username" placeholder="**************" class="input-box " required/>
```
Comment (siddhigate) [inline pages/Loginin.html]: 1. You can use `type="email"` for email inputs. You can use it for better accessibility.
2. You have used for label `for="password"` attribute but haven't mentioned the corresponding id to the input element in both login and signup page.

Everything else looks good to me🙌

**Review by siddhigate — APPROVED** [review]

File: pages/Loginin.html (hunk 3 above)
Comment (Manju-sree) [reply pages/Loginin.html]: thanks for pointing out this.. will make changes accordingly

**Pull request merged**